### PR TITLE
Add release notes for v0.3.13

### PR DESCRIPTION
**Pull request by peterschwarz**

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,59 @@
...
+### Highlights
+
+* Breaking change: Socket-based transports (TCP and TLS) now require a version
+  handshake when connecting.
+* New experimental feature `circuit-template`: Circuit template library to
+  simplify circuit creation.
+
+### Deprecations and Breaking Changes
+
+For upgrade information, see Upgrading to Splinter 0.3.13 from Splinter 0.3.12.
```
Comment (rbuysse) [inline RELEASE_NOTES.md]: Should this be a link?

Comment (peterschwarz) [reply RELEASE_NOTES.md]: Fixed

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,5 +1,59 @@
...
+  provides library code for building tools to create circuits using templates.
+* All experimental Biome features:
+    - New OpenAPI documentation for REST API routes.
+    - Improved Rust documentation; see
+      [docs.rs/splinter](https://docs.rs/splinter/).
+    - Restructured data stores to follow new model that enables future
+      database selections.
+* Experimental feature "biome-key-management": Added response bodies to the
+  key management REST API routes.
+* Experimental feature"biome-user":
```
Comment (ltseeley) [inline RELEASE_NOTES.md]: s/feature"biome-user"/feature "biome-user"/

Comment (peterschwarz) [reply RELEASE_NOTES.md]: Fixed

**Pull request merged**